exporters/elasticsearch/include/opentelemetry/exporters/elasticsearch/es_log_recordable.h

```diff
@@ -201,27 +202,42 @@ class ElasticSearchRecordable final : public sdk::logs::Recordable
   nlohmann::json GetJSON() noexcept { return json_; }
 
   /**
-   * Set instrumentation_library for this log.
-   * @param instrumentation_library the instrumentation library to set
+   * Set instrumentation_scope for this log.
+   * @param instrumentation_scopehe instrumentation library to set
    */
+  void SetInstrumentationScope(const opentelemetry::sdk::instrumentationscope::InstrumentationScope
+                                   &instrumentation_scope) noexcept
+  {
+    json_["name"]          = instrumentation_scope.GetName();
+    instrumentation_scope_ = &instrumentation_scope;
+  }
+
+  OPENTELEMETRY_DEPRECATED_MESSAGE("Please use SetInstrumentationScope instead")
   void SetInstrumentationLibrary(
-      const opentelemetry::sdk::instrumentationlibrary::InstrumentationLibrary
-          &instrumentation_library) noexcept
+      const opentelemetry::sdk::instrumentationscope::InstrumentationScope
+          &instrumentation_scope) noexcept
+  {
+    SetInstrumentationScope(instrumentation_scope);
+  }
```

Comment:
Do we need this method `SetInstrumentationLibrary` - as it is already defined it `sdk::logs::Recordable`?

Reply:
Thanks, and this can be removed.